data/dictionary/items getter/internal/b6d7b23b-3795-4dbc-bee0-ffd139641f13.json

```diff
@@ -28,7 +32,7 @@
       "dependencies": {},
       "initFunction": "",
       "inputFunctions": {
-        "9af21210-8f9c-4067-986a-f29fe11cede0": "const {dict, keys} = data;\noutputs.values(keys.map(key => dict[key]), tag);"
+        "9af21210-8f9c-4067-986a-f29fe11cede0": "const {dict, keys} = data;\n\nconst result = keys.reduce((previousResults, currentKey) => {\n  const value = dict[currentKey];\n  \n  if (value === undefined) {\n     previousResults[1].push(currentKey);\n     previousResults[0].push(null);\n  } else {\n    previousResults[0].push(value);\n  } \n  \n return previousResults;\n}, [[],[]]);\n\noutputs.values(result[0], tag);\noutputs[\"not found keys\"](result[1], tag);\n"
```

Comment:
I think this node's name is not appropriate anymore as it is only used in `data/dictionary/Values getter by keys`

Reply:
Done